feature-examples/python/create_apikey.py

```diff
@@ -0,0 +1,129 @@
+"""Example of using the Vectara REST API to create an API Key."""
+
+import argparse
+import logging
+import requests
+import sys
+
+import vectara_util
+
+
+def create_apikey(
+    customer_id: int,
+    corpus_id: int,
+    auth_url: str,
+    app_client_id: str,
+    app_client_secret: str,
+) -> (any, bool):
+    """Creates an API key.
+
+    Args:
+        customer_id: Unique customer ID in vectara platform.
+        corpus_id: Corpus ID to which API key will be created.
+        auth_url: The cognito auth url for this customer.
+        app_client_id: The id of the app client with enough rights.
+        app_client_secret: The app client secret.
+
+    Returns:
+        (response, True) in case of success and returns (error, False) in case of failure.
```

Comment:
Please clarify what `response` is. `(apikey, True)`?

Reply:
Done.